Components/Hero.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+
+import '../css/Components/Hero'
+
+import Title from './Title';
+import SubSectionTitle from './SubSectionTitle';
+
+import PlayIcon from '../images/play.svg';
+import TwitterLogo from '../images/twitter.svg';
+import YoutubeLogo from '../images/youtube.svg';
+import FacebookLogo from '../images/facebook.svg';
+
+export default class Hero extends Component {
+  render() {
+    return <section className="Hero">
+      <div className="Hero-background">
+      </div>
+
+      <div className="Hero-foreground">
+        <div className="Hero-content">
+          <div className="Hero-title">
+            <SubSectionTitle>Mirror Conference 2.0</SubSectionTitle>
+            <div className="Hero-uvp">
+              <Title alternate>For designers and front-end developers.</Title>
```

Comment:
Irrelevant since I'm going to refactor this into a `Hero` specific child component.